### PR TITLE
Update compliance-manager-setup.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes per #https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9521 as the report we get doesn't actually include details of work done and no notes exist as part of the export

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/compliance-manager-setup.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ The global administrator for your organization can change the settings for autom
 
 ### Manage user history
 
-The **Manage user history** settings help you quickly identify which users have worked with improvement actions in Compliance Manager. The identifiable user data associated with improvement actions includes any implementation and testing work done, documents they uploaded, and any notes they entered. Understanding and retrieving this type of data may be necessary for your organization’s own compliance needs.
+The **Manage user history** settings help you quickly identify which users have worked with improvement actions in Compliance Manager. The identifiable user data associated with improvement actions includes the status of the Improvement actions and  documents they uploaded. Understanding and retrieving this type of data may be necessary for your organization’s own compliance needs.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-setup.md]:
```suggestion
The **Manage user history** settings help you quickly identify which users have worked with improvement actions in Compliance Manager. The identifiable user data associated with improvement actions includes the status of the improvement actions and  documents they uploaded. Understanding and retrieving this type of data may be necessary for your organization’s own compliance needs.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chvukosw Please merge this PR. Thanks

**Pull request merged**